server/php/UploadHandler.php

```diff
@@ -131,19 +131,21 @@ public function __construct($options = null, $initialize = true, $error_messages
             // Command or path for to the ImageMagick identify binary:
             'identify_bin' => 'identify',
             'image_versions' => array(
-                // The empty image version key defines options for the original image:
-                // Keep in mind that these options are inherited by all other image versions!
+                // The empty image version key defines options for the original image.
+                // Keep in mind: these image manipulations are inherited by all other image versions from this point onwards. 
+                // Also note that the property 'no_cache' is not inherited, since it's not a manipulation.
                 '' => array(
                     // Automatically rotate images based on EXIF meta data:
                     'auto_orient' => true
                 ),
-                // Uncomment the following to create medium sized images:
+                // You can add an array. The key is the name of the version (example: 'medium'). 
+                // the array contains the options to apply.
                 /*
                 'medium' => array(
-                    'max_width' => 800,
-                    'max_height' => 600
-                ),
-                */
+                    'max_width' => 200,
+                    'max_height' => 160
+                ), 
+		*/
```

Comment:
Again, please keep the original dimensions for the example.
Changing sample dimensions should only be done if there's a good reason for it.